_sass/addon/commons.scss

```
@@ -959,6 +959,7 @@ $btn-mb: 0.5rem;
   color: var(--link-color);
   margin-left: 1rem;
```

Alright, please ignore the modification request above. Keep your changes, and add a margin improvement:

```suggestion
  margin-left: 0.75rem;
```

---

ok